.github/workflows/update.yml

```diff
@@ -29,12 +29,10 @@ jobs:
     - name: Fetch followers
       run: go run . -- followers > ../followers.txt
     - name: Commit results
-      run: |
-        set -e
-        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-        git config --global user.name "github-actions[bot]"
-
-        d="$(date --rfc-3339=s)"
-        git add --all
-        git commit --allow-empty -m "Periodic update: $d"
-        git push origin master
+      uses: EndBug/add-and-commit@v9
+      with:
+        committer_name: GitHub Actions
+        committer_email: 41898282+github-actions[bot]@users.noreply.github.com
+        fetch: false
+        message: update-${{ env.GITHUB_RUN_NUMBER }}
```

Comment:
Would be good to retain the commit msg if GitHub offers a date variable.

Reply:
Doesn't look like timestamp or such is an option but you can still use the same way of setting the message. I thought each commit has a date on it anyway then the sequential number was more intuitive. Either way is fine.

Reply:
Apparently there's a way but it is rather complicated. https://stackoverflow.com/a/60942437

It's ok for now. You tested this and it works right?

Reply:
Works in my repo with my token ;)